### DISCUSSION TITLE
stability of selkies desktop

### DISCUSSION THREAD

**Discussion by jose-d** [selkies-project/selkies-gstreamer#106] [General]
What is the general stability of selkies web desktop session?

It seems to me that when I keep the session in the background, every then and now it freezes.

Is it something known, or something possibly related to my infrastructure?

Currently I toy with the provided nvidia docker egl example against Firefox.

**Comment by ehfd** [comment]
It's possible that your browser's tab sleep mode is activated. Refreshing solves the issue if so.

Otherwise, it is possible that the WebRTC protocol has configurations that should be further optimized.

**Reply by ehfd** [reply]
Fixed in next release. #78
